controllers/rhmi/bootstrapReconciler.go

```diff
@@ -113,6 +113,17 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 	}
 	// MGDAPI-4641 block end
 
+	operatorLimitRangeParams := resources.LimitRangeParams{
+		CpuRequest:    "5m",
+		MemoryRequest: "10Mi",
+		ResourceType:  corev1.LimitTypeContainer,
+	}
+	phase, err = resources.ReconcileLimitRange(ctx, serverClient, r.installation.Namespace, operatorLimitRangeParams)
```

Comment:
```suggestion
	phase, err = resources.ReconcileLimitRange(ctx, serverClient, r.installation.Namespace, resources.DefaultLimitRangeParams)
```
Example if you adapt to use the exported parameter 🤔